bolt-sidecar/src/primitives/commitment.rs

```diff
@@ -143,12 +143,23 @@ impl InclusionRequest {
         true
     }
 
-    pub fn validate_priority_fee(&self) -> bool {
+    pub fn validate_max_priority_fee(&self) -> bool {
         for tx in &self.txs {
-            if tx
-                .max_priority_fee_per_gas()
-                .is_some_and(|max_priority_fee| max_priority_fee > tx.max_fee_per_gas())
-            {
+            let gas_price = tx.as_legacy().map(|tx| tx.gas_price).unwrap_or(0);
+            let max_priority_fee = tx.max_priority_fee_per_gas().unwrap_or(0);
+            if max_priority_fee + gas_price > tx.max_fee_per_gas() {
+                return false;
+            }
```

Comment:
I don't fully get this, what does `tx.max_fee_per_gas` have to do with this?

Reply:
btw this was already there though